lms/djangoapps/certificates/views.py

```diff
@@ -481,9 +468,19 @@ def render_html_view(request, user_id, course_id):
     configuration = CertificateHtmlViewConfiguration.get_config()
     context.update(configuration.get('default', {}))
 
+    # Translators:  This is the copyright line which appears at the bottom of the certificate page/screen
+    context['copyright_text'] = _('&copy; {year} {platform_name}. All rights reserved.').format(
```

Comment:
As you recommended below, I'll add a note to the translators comment about the escape code.  For the actual i18n, how about something like:

```python
rights = _("All rights reserved.")
context['copyright_text'] = '&copy; {year} {platform_name}. {reserved}'.format(...)
```